src/custom/simplifions/components/SimplifionsCasDusageDescription.vue

```diff
@@ -24,22 +24,22 @@
             Sommaire
           </h2>
           <ol>
-            <li>
+            <li v-if="casUsage.Contexte || casUsage.Cadre_juridique">
               <a
                 id="summary-link-1"
                 href="#contexte-et-cadre-juridique"
                 class="fr-summary__link"
                 >Contexte et cadre juridique</a
               >
             </li>
-            <li>
+            <li v-if="casUsage.reco_solutions?.length">
```

Comment:
:sparkles:

Reply:
What ? Sparkles ? xD
(I resolve this comment)

Reply:
Yeah, just found it nice :p